<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export_KML</class>
 <widget class="QWidget" name="Export_KML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导出到Google Earth</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/GoogleEarth.png</normaloff>:/QtWidgetsApplication3/bin/icon/GoogleEarth.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>目标工程</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>数据节点</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>保存文件名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="File_path"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>保存位置</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="File_name"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="Browse">
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Export">
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
